cluster/examples/coreos/before-reboot-daemonset.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-node-annotator
+  namespace: rook
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: rook
+  name: rook-node-annotator
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["patch"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-node-annotator
+  namespace: rook
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-node-annotator
+subjects:
+- kind: ServiceAccount
+  name: rook-node-annotator
+  namespace: rook
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: ceph-reboot-check
+  namespace: rook
+spec:
+  template:
+    metadata:
+      labels:
+        app: ceph-reboot-check
+    spec:
+      serviceAccountName: rook-node-annotator
+      nodeSelector:
+        container-linux-update.v1.coreos.com/before-reboot: "true"
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: ceph-reboot-check
+        image: rook/toolbox:master
+        imagePullPolicy: IfNotPresent
+        command: ["/scripts/status-check.sh"]
+        env:
+          - name: ROOK_ADMIN_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: rook-ceph-mon
+                key: admin-secret
+          - name: NODE
+            valueFrom:
+               fieldRef:
+                  fieldPath: spec.nodeName
+        securityContext:
+          privileged: true
+        volumeMounts:
```

Comment:
if this toolbox instance is going to be used for the ceph reboot script, it probably doesn't need these other mounts like `/dev` and `/sys/bus`, and it doesn't need to run as privileged either I suspect.  Those are needed for the toolbox to mount RBD devices and the like, which this won't be doing.  Is that correct?

Reply:
I'll try to delete the following volume mounts:
-  `/dev`
- `/sys/bus`

and delete the option `privileged: true`
I'll test the results as soon as I can, because I don't know wether there necessary for the script to work

Reply:
Without the volumes and privileged options, the container still shows the desired result on the `ceph health`.
I'll update the pull request!